packages/aws-lambda-otel-extension/opt/otel-extension/external/otel-payloads.js

```diff
@@ -118,6 +118,78 @@ const createResourceAttributes = (fun) =>
     }
   });
 
+const createLogPayload = (fun, logs) => {
+  const spanData = fun.span;
+  const key = Object.keys(fun.eventData)[0];
+  const metricsAtt = createMetricAttributes({ record: fun.eventData[key] }, {})
+    .filter((attr) => {
+      return [
+        'faas.arch',
+        'faas.api_gateway_request_id',
+        'faas.event_source',
+        'faas.api_gateway_app_id',
+      ].includes(attr.key);
+    })
+    .reduce(
+      (obj, attr) => ({
+        ...obj,
+        [attr.key]: Object.values(attr.value)[0],
+      }),
+      {}
+    );
+  const resourceAtt = createResourceAttributes({ record: fun.eventData[key] })
+    .filter((attr) => {
+      return [
+        'faas.id',
+        'faas.name',
+        'cloud.region',
+        'sls.app_uid',
+        'service.namespace',
+        'deployment.environment',
+        'service.name',
+        'telemetry.sdk.language',
+        'telemetry.sdk.name',
+        'telemetry.sdk.version',
+        'cloud.provider',
+        'cloud.account.id',
+        'cloud.platform',
+        'faas.collector_version',
+      ].includes(attr.key);
+    })
+    .reduce(
+      (obj, attr) => ({
+        ...obj,
+        [attr.key]: Object.values(attr.value)[0],
+      }),
+      {}
+    );
+
+  const severityNumberMap = {
+    TRACE: 1,
+    DEBUG: 5,
+    INFO: 9,
+    WARN: 13,
+    ERROR: 17,
+    FATAL: 21,
+  };
+
+  return logs.map((log) => {
+    const split = (log.record || '').split('\t');
```

Comment:
I think format as `2019-04-19T08:07:05.666Z c82d8e46-fcc6-4186-979d-2bc896c1b0b2 INFO <logMessage>` is produced only by logs as written via `console` method.

If someone in code uses `process.std*` (or some log util that uses that internally) then log will be simply as `<logMessage>`.

I assume it's better to take that into account here, to avoid weird processing issues

Reply:
This should be set up to support `process.std*`. It won't pick up the log level but we will have to make some adjustments for the logging frameworks we want to support. I don't have this list just yet but I can create a follow up PR once we know 👍

Reply:
Alright the severity fields should not be included in the request if they are not defined 👍